File: app/code-examples/crop-health-binary.js
const imageData = fs.readFileSync('cocoa.jpg');

// Get the prediction for image cocoa.jpg passed as a binary file in the request body
fetch.then(async fetch => {
    const response_binary = await fetch(
        "https://api-test.openepi.io/crop-health/predictions/binary",
        {
            method: "POST",
            body: imageData,
        }
    );
    const data_binary = await response_binary.json();
	// Print the prediction for the healthy class
    console.log(data_binary.HLT);
});

File: app/code-examples/crop-health-binary.py
from httpx import Client

with Client() as client:
    # Get the binary model prediction for image cocoa.jpg 
    # passed as a binary file in the request body
    response_binary = client.post(
        url="https://api-test.openepi.io/crop-health/predictions/binary",
        content=open("cocoa.jpg", "rb").read(),
    )

    data_binary = response_binary.json()
    # Print the prediction for the healthy class
    print(data_binary["HLT"])